styles/index.css

```diff
@@ -0,0 +1,18 @@
+.red#temp-display{
+    color: red;
+}
+.orange#temp-display{
+    color: orange;
+}
+.yellow#temp-display{
+    color: yellow;
+}
+.green#temp-display{
+    color: green;
+}
+.teal#temp-display{
+    color: teal;
+}
+
+
```

Comment:
Y'all...I love craigslist chic!! Bring back the HTML websites of the 90s! This is a solid css file 💯